engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableMaxRefreshingAggTest.java

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl;
+
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.util.TableTools;
+import org.junit.Test;
+
+public class QueryTableMaxRefreshingAggTest extends QueryTableMaxAggTestBase {
+
+    @Override
+    public Table max(char[] source) {
+        final Table x = TableTools.newTable(TableTools.charCol(S1, source));
+        x.setRefreshing(true);
+        return x.maxBy();
```

Comment:
This only tests the instantiation (addChunk) case. It doesn't test removal or modification handling. That said, we have adequate testing to compare recomputed results to updated results.